test/plugins/validation/2and3/parameters-ibm.js

```diff
@@ -299,9 +299,91 @@ describe('validation plugin - semantic - parameters-ibm', () => {
       expect(res.errors.length).toEqual(0);
       expect(res.warnings.length).toEqual(0);
     });
+
+    it('should not return an error for parameters that live in the top level', () => {
+      const spec = {
+        parameters: [
+          {
+            name: 'someparam',
+            in: 'header',
+            type: 'string',
+            required: true,
+            description: 'some top level param'
+          }
+        ],
+        paths: {
+          '/pets': {
+            get: {
+              parameters: [
+                {
+                  name: 'file',
+                  in: 'formData',
+                  type: 'file',
+                  required: true,
+                  description: 'A file passed in formData'
+                }
+              ]
+            }
+          }
+        }
+      };
+
+      const res = validate({ jsSpec: spec, isOAS3: false }, config);
+      expect(res.errors.length).toEqual(0);
+      expect(res.warnings.length).toEqual(0);
+    });
```

Comment:
I think you can remove this test actually, I don't think it adds too much. I appreciate the robustness though